crates/types/Cargo.toml

```diff
@@ -8,3 +8,6 @@ aws-sdk-dynamodb = {workspace = true}
 serde = {workspace = true}
 thiserror = {worksapce = true}
 uuid = {workspace = true}
+
+[dev-dependencies]
+serde_json = {workspace = true}
```

Comment:
I would still change it to `serde_json.workspace = true`, it's more modern way, I've asked Denis about that too, and he confirmed that